bin/moveAndCheckSamplesheets.sh

```diff
@@ -192,41 +185,111 @@ samplesheetsSource="${DAT_ROOT_DIR}/samplesheets/new/"
+		cp "${samplesheet}"{,.converted}
+	
+		#
+		# Make sure
+		#  1. The last line ends with a line end character.
+		#  2. We have the right line end character: convert any carriage return (\r) to newline (\n).
+		#  3. We remove empty lines.
+		#
+		printf '\n'     >> "${samplesheet}.converted"
+		sed 's/\r/\n/g' "${samplesheet}.converted" > "${samplesheet}.converted.tmp"
+		sed '/^\s*$/d'  "${samplesheet}.converted.tmp" > "${samplesheet}.converted.tmp2"
+		rm "${samplesheet}.converted.tmp"
+		mv "${samplesheet}.converted.tmp2" "${samplesheet}.converted"
+		if checkSampleSheet.py --input "${samplesheet}.converted" --log "${samplesheet}.converted.log"
+		then
+			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "samplesheet is correct"
+			check=$(cat "${samplesheet}.converted.log")
+		else
+			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "samplesheet is not correct"
+			check=$(cat "${samplesheet}.converted.log")
+		fi
+		projectSamplesheet="false"
+		if [[ "${check}" == *'OKAY'* ]]
+		then
+			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Samplesheet is correct, continue"
+			if [[ "${check}" == *'projectSamplesheet'* ]]
+			then
+				projectSamplesheet="true"
+			fi
+				
+		else
+			log4Bash 'WARN' "${LINENO}" "${FUNCNAME:-main}" '0' "${check} for samplesheet: ${samplesheet}"
+			mv -v "${JOB_CONTROLE_FILE_BASE}."{started,failed}
+			continue
+		fi
+	fi
+	
 	if [[ -n "${_sampleSheetColumnOffsets["${PIPELINECOLUMN}"]+isset}" ]] 
 	then
+		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "column [${PIPELINECOLUMN}] is found in the samplesheet"
 		_pipelineFieldIndex=$((${_sampleSheetColumnOffsets["${PIPELINECOLUMN}"]} + 1))
 		## In future this valueInSamplesheet will be replaced by DARWIN to the real value.
 		readarray -t valueInSamplesheet < <(tail -n +2 "${samplesheet}" | cut -d "${SAMPLESHEET_SEP}" -f "${_pipelineFieldIndex}" | sort | uniq )
+		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "renaming ${valueInSamplesheet[0]} into ${REPLACEDPIPELINECOLUMN}"
 		perl -p -e "s|${valueInSamplesheet[0]}|${REPLACEDPIPELINECOLUMN}|" "${samplesheet}" > "${samplesheet}.tmp"
 		mv "${samplesheet}.tmp" "${samplesheet}"
 	else
 		awk -v pipeline="${REPLACEDPIPELINECOLUMN}" -v pipelineColumn="${PIPELINECOLUMN}" 'BEGIN {FS=","}{if (NR==1){print $0",pipelineColumn}{else print $0","pipeline}'
 	fi
 	firstStepOfPipeline="${REPLACEDPIPELINECOLUMN%%+*}"
-	#shellcheck disable=SC2153
-	samplesheetsDestination="${HOSTNAME_TMP}:/groups/${GROUP}/${SCR_LFS}/Samplesheets/${firstStepOfPipeline}/new/"
+
+
+	# Check whether the samplesheet is a project samplesheet (no NGS_Demultiplexing)
+	# needs an update when we are going to use VIP into production
+	if [[ ${projectSamplesheet} == "true" ]]
+	then
+		firstStepOfPipeline="NGS_DNA"
+		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "the samplesheet is a project samplesheet (no NGS_Demultiplexing), samplesheet moved to ${firstStepOfPipeline}"
```

Comment:
```suggestion
		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "The samplesheet is a project samplesheet (no NGS_Demultiplexing); firstStepOfPipeline was set to ${firstStepOfPipeline}."
```
Note that I do not this moving of the samplesheet